varianteval/scripts/annotation_script.py

```diff
@@ -0,0 +1,136 @@
+"""
+Script to add annotation to input callset with respect to specified
+comparison callsets
+"""
+
+import sys
+import re
+import numpy as np
+from pysam import VariantFile
+import argparse
+from collections import defaultdict
+from intervaltree import IntervalTree
+
+
+def overlapping_intervals(intrvl_A, intrvl_B, overlap_threshold=0.5, bp_margin=5000):
+    '''
+    Helper method to determine whether two records' start-end
+    intervals overlap within a given breakpoint margin, and whether
+    the intervals' overlap is >=0.5*(svlen) for interval A and B
```

Comment:
overlap_threshold instead of 0.5 in the comment

Reply:
Done